content/800-data-platform/100-accelerate/500-limitations.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Limitations'
+metaTitle: 'Accelerate: Limitations'
+metaDescription: 'Learn about limitations of Accelerate.'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Below are descriptions of known limitations when using Accelerate. If you are aware of any limitations that are missing, please let us know on the [#accelerate-feedback](https://prisma.slack.com/archives/C04KTN0V2Q0) channel in our community Slack.
+
+</TopBlock>
+
+## Cannot cache raw queries
+
+At the moment, it is not possible to cache the responses of [raw queries](/concepts/components/prisma-client/raw-database-access).
+
+## Not compatible with the fluent API
+
+Client Extensions (which are used in Accelerate) currently do not correctly forward the [fluent API](/concepts/components/prisma-client/relation-queries#fluent-api) types. We hope to get a fix into Client Extensions soon.
+
+## Query timeout limit
+
+Accelerate has a global timeout of `10s` for each query. Reach out to [support@prisma.io](mailto:support@prisma.io) with your use case if your application requires a greater timeout value.
+
+## Interactive transactions query timeout limit
+
+Accelerate has a global timeout of `15s` for each [interactive transaction](/guides/performance-and-optimization/prisma-client-transactions-guide#interactive-transactions). Reach out to [support@prisma.io](mailto:support@prisma.io) with your use case if your application requires a greater timeout value.
+
+## Response size limit
+
+Accelerate has a global response size limit of `5MB`. Reach out to [support@prisma.io](mailto:support@prisma.io) with your use case if your application requires a larger response size.
```

Comment:
These feel more important than the raw query and fluent API limitations. I suggest to move them up.